iphone/cli/commands/_buildModule.js

```diff
@@ -915,12 +915,11 @@ iOSModuleBuilder.prototype.packageModule = function packageModule(next) {
 
 		// 3. platform folder
 		if (fs.existsSync(this.platformDir)) {
-			this.dirWalker(this.platformDir, function (file, name) {
-				var stat = fs.statSync(file);
-				if (name !== 'README.md') {
-					dest.append(fs.createReadStream(file), { name: path.join(moduleFolders, 'platform', path.relative(this.platformDir, file)), mode: stat.mode });
-				}
-			}.bind(this));
+			dest.directory(
+				this.platformDir,
+				path.join(moduleFolders, 'platform'),
+				(entryData) => entryData.name === 'README.md' ? false : entryData
```

Comment:
Bah, linting doesn't like the syntax I suggested. I suppose you either need to wrap the return value in parens or use curly braces with a `return`.

```suggestion
				(entryData) => (entryData.name === 'README.md' ? false : entryData)
```

or if needed:

```
				(entryData) => { return entryData.name === 'README.md' ? false : entryData; }
```

Reply:
Yep, that one helped. VSCode shows it locally as well